src/pymor/reductors/mt.py

```diff
@@ -0,0 +1,237 @@
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright 2013-2020 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+import numpy as np
+import scipy.linalg as spla
+from pymor.algorithms.gram_schmidt import gram_schmidt, gram_schmidt_biorth
+from pymor.algorithms.samdp import samdp
+from pymor.algorithms.to_matrix import to_matrix
+from pymor.core.base import BasicObject
+from pymor.models.iosys import LTIModel, sparse_min_size
+from pymor.operators.constructions import IdentityOperator
+from pymor.operators.numpy import NumpyMatrixOperator
+from pymor.parameters.base import Mu
+from pymor.reductors.basic import LTIPGReductor
+
+
+class MTReductor(BasicObject):
+    """Modal Truncation reductor.
+
+    See Section 9.2 in [A05]_.
+
+    Parameters
+    ----------
+    fom
+        The full-order |LTIModel| to reduce.
+    mu
+        |Parameter values|.
+    """
+
+    def __init__(self, fom, mu=None):
+        assert isinstance(fom, LTIModel)
+        if not isinstance(mu, Mu):
+            mu = fom.parameters.parse(mu)
+        assert fom.parameters.assert_compatible(mu)
+        self.fom = fom
+        self.mu = mu
+        self.V = None
+        self.W = None
+        self._pg_reductor = None
+
+    def reduce(self, r=None, decomposition='samdp', projection='orth',
+               symmetric=False, which="NR", method_options=None, take_complex=False):
```

Comment:
```suggestion
               symmetric=False, which='NR', method_options=None, take_complex=False):
```

It is more "pyMOR-ic" to use single quotes for strings, unless the string itself contains single quotes.

A similar comment applies to a few places below.